x/nftmarket/keeper/utils.go

```diff
@@ -8,3 +14,26 @@ func Contains(a []string, x string) bool {
 	}
 	return false
 }
+
+func (k Keeper) GetMemo(ctx sdk.Context) (string, error) {
+	txBytes := ctx.TxBytes()
```

Comment:
Have you tested this behaviour via CLI?

Reply:
yes.

Reply:
Can you add example CLI commands when test so that other people could test with same command. It will be time taking to compose CLI commands again. (Just recommendation)